disclosure/templates/missing-ballot-data.html

```diff
@@ -0,0 +1,8 @@
+{% if bad_ballots %}
+  <h2>Unedited, auto-generated ballots:</h2>
+  <ul>
+  {% for bad_ballot in bad_ballots %}
+    <li><a href="/admin/ballot/ballot/{{ bad_ballot.id }}/">{{ bad_ballot }}</a></li>
+  {% endfor %}
+  </ul>
+{% endif %}
```

Comment:
Message for when there is no bad data?

It's always confusing when you get empty results and you're not sure if there was an error or what.